streams/src/test/java/org/apache/kafka/streams/processor/internals/InternalTopicManagerTest.java

```diff
@@ -153,7 +156,8 @@ public void shouldNotCreateTopicsWithEmptyInput() throws Exception {
 
     @Test
     public void shouldOnlyRetryNotSuccessfulFuturesDuringSetup() {
-        final AdminClient admin = EasyMock.createNiceMock(AdminClient.class);
+        final AdminClient admin = EasyMock.createStrictMock(AdminClient.class);
+        config.put(ConsumerConfig.MAX_POLL_INTERVAL_MS_CONFIG, 10_000L);
```

Comment:
In test setup, we reduce max.poll to 100ms, to increasing it to 10 sec for this test should be sufficient.

Reply:
I think a default mock would be enough to get a better error message. Strict mocks also check the order of the calls which is not needed here. 

I would also pass `new MockTime(1)` instead of increasing the timeout, because the test retrieves the time twice. `new MockTime(1)` would progress time 1 ms each time. Hence, we would be within the 100 ms of the global `max.poll.interval.ms` and since we control the time progression the test would be more robust than by increasing `max.poll.interval.ms` to a higher value. With `new MockTime(1)`, we also avoid running the test indefinitely in case of a mistake in the production code, since after 50 time retrievals, the test would run into a timeout and the admin client mock would throw the unexpected method call error. Probably, it would be even better to use something like `new MockTime((Integer) config.get(StreamsConfig.consumerPrefix(ConsumerConfig.MAX_POLL_INTERVAL_MS_CONFIG)) / 3)`.

Reply:
After this PR is merged, I will open a PR that changes also the other usages of `Time.SYSTEM` and `createNiceMock()` since I think that would make all tests more robust. I do not do it now because I am still working on that code for KIP-698 and I do not want to run into too many merge conflicts.

Reply:
Cool. Works for me.